app/errors.py

```diff
@@ -0,0 +1,16 @@
+class VaccineError(Exception):
+    def __init__(self, *args) -> None:
+        super().__init__(*args)
+
+
+class NotVaccinatedError(VaccineError):
+    pass
```

Comment:
Consider adding a custom error message to the `NotVaccinatedError` class to provide more context when this exception is raised. This can help users understand the specific issue without needing to refer to the documentation.